articles/storage/common/storage-network-security.md

```diff
@@ -90,7 +90,11 @@ After you apply network rules, they're enforced for all requests. SAS tokens tha
 
 [Network Security Perimeter](../../private-link/network-security-perimeter-concepts.md) (preview) allows organizations to define a logical network isolation boundary for PaaS resources (for example, Azure Blob Storage and SQL Database) that are deployed outside their virtual networks. The feature restricts public network access to PaaS resources outside the perimeter. However, you can exempt access by using explicit access rules for public inbound and outbound traffic. By design, access to a storage account from within a Network Security Perimeter takes the highest precedence over other network access restrictions.
 
-Currently, Network Security Perimeter is in public preview for Azure Blobs, Azure Files (REST), Azure Tables, and Azure Queues. See [Transition to a Network Security Perimeter](../../private-link/network-security-perimeter-transition.md).
+Currently, Network Security Perimeter is in public preview for Azure Blobs, Azure Files (REST), Azure Tables, and Azure Queues. See [Transition to a Network Security Perimeter](../../private-link/network-security-perimeter-transition.md). 
+
+The list of services that have been onboarded to NSP can be found [here](../../private-link/network-security-perimeter-concepts#onboarded-private-link-resources). 
```

Comment:
```suggestion
The list of services that have been onboarded to NSP can be found [here](../../private-link/network-security-perimeter-concepts.md#onboarded-private-link-resources). 
```

Reply:
Missing the "md" file extension here.

Reply:
@Srikumar-vaitinadin - if you accept this suggestion, one of the validation errors go away. You need the "md" file extension here.